libs/coin-modules/coin-aptos/src/test/cli.ts

```diff
@@ -0,0 +1,44 @@
+import invariant from "invariant";
+import type { AccountLike } from "@ledgerhq/types-live";
+import type { Transaction } from "../types";
+import BigNumber from "bignumber.js";
+
+const options = [
+  {
+    name: "fee",
+    type: String,
+    desc: "how much fee",
+  },
+  {
+    name: "tag",
+    type: Number,
+    desc: "aptos tag",
+  },
+];
```

Comment:
Honest question: how do we know which options we need to add here?

Reply:
Honest answer: no clue 😄 
I essentially just copied from another coin and changed where it made sense. We need to have this validated by @hedi-edelbloute whenever he's back (which I'd say, doesn't block the merge).

Reply:
I think you should keep this logic inside `ledger-live-common/src/families/aptos` Then we will move all the logic inside the coin-module (during the coin modularization)